colorizer_data/converter.py

```diff
@@ -81,15 +87,8 @@ def _make_frame(
     seg2d = scale_image(seg2d, scale)
     seg2d = seg2d.astype(np.uint32)
 
-    # Remap the frame image so the IDs are unique across the whole dataset.
-    seg_remapped, lut = remap_segmented_image(
-        seg2d,
-        frame,
-        config.object_id_column,
-    )
```

Comment:
IDs no longer have to be unique across the dataset, so this step was eliminated entirely.